akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsSampleSpec.scala

```diff
@@ -45,7 +45,7 @@ object StatsSampleSpecConfig extends MultiNodeConfig {
           cluster {
             enabled = on
             allow-local-routees = on
-            use-role = compute
+            use-role-set = ["compute"]
```

Comment:
yes, kind of. We'll help you with coming up with some workaround (there are ugly tricks).